duckduckgo/Android
# Add maestro to CI

Task/Issue URL: https://app.asana.com/0/1202552961248957/1202846016452961/f

### Description
This PR integrates maestro tests with CI to run nightly. For now we only run ad click tests.

### Steps to test this PR
Tests should pass, see https://github.com/duckduckgo/Android/actions/runs/3640577449 for latest run
